### PR TITLE
Add TTL to SIEM Lookup API (in bulk add elements)

### PR DESCRIPTION
**Pull request by YajB**
# What changed

Descriptions for both TTL & expirationDate: both params are optional. "When this parameter is left empty, the lookup list does not expire."

Bulk add: https://deploy-preview-1070--logz-docs.netlify.app/api/#operation/addLookupListElements

[Add an element was affected because a definition changed :   https://deploy-preview-1070--logz-docs.netlify.app/api/#operation/createLookupListElement]

Only one POST method needed to have TTL parameter added: 
- /lookup-lists/{lookupListId}/bulk-add:  https://deploy-preview-1070--logz-docs.netlify.app/api/#operation/addLookupListElements

**In VSC:** 
+ ` /v1/lookup-lists/{lookupListId}/elements`:   approximately line 3426-3458
+ ` /v1/lookup-lists/{lookupListId}/elements/bulk-add:`:    approximately line 3578-3618
+ both endpoints reference the following definition `/LookupListElementCreateRequest'`: approximately line 3657-3677: added the `expirationDate`

`/v1/lookup-lists/{lookupListId}/elements`:  Line 3436 - Parameters section, removed
```
        - name: accountID
          in: query
          required: false
          type: integer
          format: int32
          description:  Logz.io account ID
```

----

## Pages to review

- LinkToPage1
- LinkToPage2

## Remaining work

@ohadza :
- [x] Done: What is the default value of TTL? Empty. Also expirationDate
- [x] Done: How can a user configure the list so that it never expires?  leave the param value empty (TTL & expirationDate
- [ ] **TBD**: Which value takes precedence: TTL or expirationDate. (one is in query, one is in request body schema)

## Post launch

To be completed by the docs team upon merge:

- [ ] Teams to update with the new information:
- [ ] Replace original content on the support portal with 'this page has been moved to ...' - paste the URL
- [ ] Update these log shipping pages in the app: - paste the URL

### CODE REVIEW

File: _source/api/logzio-public-api.yml [hunk 1]
```diff
@@ -3575,10 +3581,22 @@ paths:
...
       produces:
         - application/json
       parameters:
+        - name: accountId
+          in: query
+          required: false
+          type: integer
+          format: int32
+          description:  Logz.io account ID
+        - name: defaultTTL
```
Comment (YajB) [inline _source/api/logzio-public-api.yml:3594]: @ohadza 

 What is the default value of TTL?
 How can a user configure the list so that it never expires?

Comment (ohadza) [reply _source/api/logzio-public-api.yml:3594]: the default for defaultTTL is empty it also is never expires
for expirationTime null is never expires
BTW I consider adding to implementation that 0 also never expires, what do you think?

File: _source/api/logzio-public-api.yml [hunk 2]
```diff
@@ -3433,6 +3433,12 @@ paths:
       produces:
         - application/json
       parameters:
+        - name: accountID
+          in: query
+          required: false
+          type: integer
+          format: int32
+          description:  Logz.io account ID
```
Comment (ohadza) [inline _source/api/logzio-public-api.yml:3441]: pls remove that

File: _source/api/logzio-public-api.yml [hunk 3]
```diff
@@ -3575,10 +3581,22 @@ paths:
...
+      description: Adds an array of elements to an existing Lookup list and sets the expiration date for the lookup.
       produces:
         - application/json
       parameters:
+        - name: accountId
+          in: query
+          required: false
+          type: integer
+          format: int32
+          description:  Logz.io account ID
```
Comment (ohadza) [inline _source/api/logzio-public-api.yml:3593]: pls remove that

File: _source/api/logzio-public-api.yml [hunk 4]
```diff
@@ -3587,7 +3605,7 @@ paths:
           example: 7c985e09-3db6-5dc6-ae33-58403493e13f
         - in: body
           name: body
-          required: true
+          required: false
```
Comment (ohadza) [inline _source/api/logzio-public-api.yml:3608]: why did you change that to not required?

**Review by ohadza — CHANGES_REQUESTED** [review]

File: _source/api/logzio-public-api.yml (hunk 1 above)
Comment (ohadza) [reply _source/api/logzio-public-api.yml:3594]: after consulting with Yasmin I leave the implementation as it right now
Meaning no 0

File: _source/api/logzio-public-api.yml (hunk 2 above)
Comment (YajB) [reply _source/api/logzio-public-api.yml]: Removed the accountID field

File: _source/api/logzio-public-api.yml (hunk 3 above)
Comment (ohadza) [reply _source/api/logzio-public-api.yml:3587]: 👆
